staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -34,6 +35,13 @@ import (
 	"k8s.io/apiserver/pkg/storage/value"
 )
 
+func init() {
```

Comment:
*really* not a fan of doing this in an init block... can we do something in a once block when we construct etcd3 clients?

Reply:
I will look into it.

Reply:
Ok, Then I have to use sync once in `newETCD3Client`